src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php

```diff
@@ -77,9 +77,24 @@ public function getContentPath($url, $webspaceKey = null, $locale = null)
                 $locale ?: $this->requestAnalyzer->getCurrentLocalization()->getLocalization(),
                 $webspaceKey
             );
-
             if (sizeof($portalUrls) > 0) {
-                return rtrim($portalUrls[0], '/');
+                if ($domain) {
```

Comment:
I think the same, this logic should probably also be useable in a non-twig context. Don't really know what you want to say with the usage...